### ISSUE TITLE
Widgets cannot be removed, edited, or activated by keyboard.

### ISSUE THREAD

**Issue by dankeck** [Hiebeler/socialfolio-frontend#14]
When a widget gets keyboard focus, two little buttons appear at the top:

![Mastodon widget with a red X button on the top left and a black edit button on the top right](https://github.com/user-attachments/assets/0db69e21-89f8-488a-8fdc-5f88b73328e7)

These buttons can be activated by mouse or touch, but not by keyboard. The same is true for another user trying to activate a widget and go to the user's social media page. 

The "Add widget" widget also cannot be activated by keyboard.

**Comment by Hiebeler** [comment]
This should be fixed with this commit, The hole accessibility via the keyboard for the hole website is now improved.

**Comment by Hiebeler** [comment]
Of course, I just added it.
